index.js

```diff
@@ -1,32 +1,67 @@
-module.exports = {
-  extends: [
-    'eslint:all',
-    'plugin:@typescript-eslint/all',
-    'plugin:import/typescript',
-    'plugin:jest/all',
-    './rules/array-func.js',
-    './rules/eslint-comments.js',
-    './rules/eslint.js',
-    './rules/functional.js',
-    './rules/import.js',
-    './rules/jest.js',
-    './rules/simple-import-sort.js',
-    './rules/sort-keys-fix.js',
-    './rules/typescript.js',
-    './rules/unicorn.js',
-    'prettier',
-    './rules/overrides.js',
-  ],
-  plugins: [
-    '@typescript-eslint',
-    'array-func',
-    'eslint-comments',
-    'functional',
-    'import',
-    'jest',
-    'simple-import-sort',
-    'sort-keys-fix',
-    'unicorn',
-  ],
-  parser: '@typescript-eslint/parser',
-};
+const typescriptEslint = require('typescript-eslint');
+const arrayFunc = require('eslint-plugin-array-func');
+const eslintComments = require('eslint-plugin-eslint-comments');
+const functional = require('eslint-plugin-functional');
+const _import = require('eslint-plugin-import');
+const jestPlugin = require('eslint-plugin-jest');
+const simpleImportSort = require('eslint-plugin-simple-import-sort');
+const unicorn = require('eslint-plugin-unicorn');
+const globals = require('globals');
+const js = require('@eslint/js');
+const eslintConfigPrettier = require('eslint-config-prettier');
+
+const arrayFuncConfig = require('./rules/array-func');
+const eslintConfig = require('./rules/eslint');
+const eslintCommentsConfig = require('./rules/eslint-comments');
+const functionalConfig = require('./rules/functional');
+const importConfig = require('./rules/import');
+const jestConfig = require('./rules/jest');
+const simpleImportSortConfig = require('./rules/simple-import-sort');
+const unicornConfig = require('./rules/unicorn');
+const typescriptConfig = require('./rules/typescript');
+
+module.exports = [
```

Comment:
realized that this file should actually stay, and export the configuration for use. I suppose the changed format makes this a breaking change, so I'll mark that in the commit message